src/language-js/printer-estree.js

```diff
@@ -302,6 +302,16 @@ function formatTernaryOperator(path, options, print, operatorOptions) {
   );
 }
 
+function getTypeScriptMappedTypeModifier(tokenNode, keyword) {
+  if (tokenNode.type === "TSPlusToken") {
+    return "+" + keyword;
+  } else if (tokenNode.type === "TSMinusToken") {
+    return "-" + keyword;
+  } else {
```

Comment:
```
✘  no-else-return  Unnecessary 'else' after 'return'  

  src/language-js/printer-estree.js:310:10
    } else {
            ^
✘ 1 problem (1 error, 0 warnings)
```

Reply:
@ikatyang 😂 fixed and linted the whole project locally.